### PR TITLE
DOCS-323 Some updates to SMB share how-to.

### PR DESCRIPTION
**Pull request by a-stjohn**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/en/docs/sharing/smb/smb-share.md [hunk 1]
```diff
@@ -12,25 +12,44 @@ of large or small pools of data. The brilliant design of the TrueNAS web
...
-to **Storage > Pools > ADD**. Give the share a custom name in the *Name* field.
-Otherwise, TrueNAS automatically assigns the name of the pool or dataset shared
-to the name of the SMB share. An optional *Description* can be specified
-to help explain the purpose of the share. For a more granular configuration,
-click **ADVANCED MODE**. See the web interface help text for detailed
-explanations.
+to share. Give the share a custom name in the *Name* field.
+Otherwise, TrueNAS automatically assigns the name of the pool or dataset
+shared to the name of the SMB share. An optional *Description* can be
+specified to help explain the purpose of the share. For a more granular
```
[the remark this answers is not in the document]
Comment (a-stjohn) [reply content/en/docs/sharing/smb/smb-share.md:42]: Yes, it's a feature in the UI. There's an actual dropdown field that gives a few different options with a short description of the purpose. They serve as kind of a template for preconfigured settings.

File: content/en/docs/sharing/smb/smb-share.md [hunk 2]
```diff
@@ -12,25 +12,44 @@ of large or small pools of data. The brilliant design of the TrueNAS web
...
+configuration, click **ADVANCED MODE**. See the web interface help text
```
[the remark this answers is not in the document]
Comment (a-stjohn) [reply content/en/docs/sharing/smb/smb-share.md]: No. Not an advanced usage article.

[the remark this answers is not in the document]
Comment (Mrt134) [reply content/en/docs/sharing/smb/smb-share.md]: Gotcha, opening a ticket to write an advanced SMB sharing article.

File: content/en/docs/sharing/smb/smb-share.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Mrt134) [reply content/en/docs/sharing/smb/smb-share.md:42]: So why didn't you put this explanation in your article?

**Pull request merged**